apps/vr-tests-react-components/src/stories/Popover.stories.tsx

```diff
@@ -36,7 +36,7 @@ positions.forEach(position => {
   stories.addStory(
     position as string,
     () => (
-      <Popover open positioning={position}>
+      <Popover withArrow open positioning={position}>
```

Comment:
Since the default is to not show the arrow should we remove `withArrow` here and let the "PopoverWithArrow" story to demonstrate this feature?

Reply:
If the design guideline has changed, then let's create a separate story for this

Reply:
I'm okay with leaving it as-is.